inference.py

```diff
@@ -240,7 +238,14 @@ def main(params):
     img_dir_or_csv = params['inference']['img_dir_or_csv_file']
 
     default_working_folder = Path(params['inference']['state_dict_path']).parent.joinpath(f'inference_{num_bands}bands')
-    working_folder = Path(get_key_def('working_folder', params['inference'], default_working_folder)) # TODO: remove working_folder parameter in all templates
+    working_folder = get_key_def('working_folder', params['inference'], None)
```

Comment:
I think we may need to dicuss custom working folders vs automatic creation of folders. For instance, it currently unintuitive  to share our train, val, tst Hdf5s since it exists in a folder that was automatically created. When devloper A runs training the logs and output are saved in the same directory as the datasets similarly for developer B. At this point it  becomes difficult to separate the training runs for each developer. I believe outputs and logs should be saved in a user defined path (custom working folder).